libs/openchallenges/api-description/src/components/schemas/Challenge.yaml

```diff
@@ -63,22 +57,18 @@ properties:
     description: The number of times the challenge has been starred by users.
     type: integer
     default: 0
+    minimum: 0
+    example: 100
   createdAt:
-    type: string
-    format: date-time
-    example: '2022-07-04T22:19:11Z'
+    $ref: CreationDatetime.yaml
```

Comment:
Either use (for consistency):
- `CreationDatetime` and `ModificationDatetime` OR
- `CreatedDatetime` and `ModifiedDatetime`